python/dolfinx/fem/petsc.py

```diff
@@ -1050,26 +1038,6 @@ def u(self) -> typing.Union[_Function, Iterable[_Function]]:
 # -- High-level interface for SNES ---------------------------------------
 
 
-def _ghostUpdate(x: PETSc.Vec, insert_mode: PETSc.InsertMode, scatter_mode: PETSc.ScatterMode):  # type: ignore
```

Comment:
Removing in favor of the one in `la.petsc`. This would have been the third duplicate!